Cargo.toml

```diff
@@ -35,6 +35,7 @@ members = [
   "ibc",
   "ibc-query",
   "ibc-testkit",
+  "tests-integration",
```

Comment:
```suggestion

  # internal and unpublishing
  "tests-integration",
```

Just wanted to add a comment.

Reply:
[9a4abfd](https://github.com/cosmos/ibc-rs/pull/1233/commits/9a4abfd695bafa34729aa6aceff8bbf654116844)